spec/spec_helper.rb

```diff
@@ -83,14 +80,21 @@ def peach_with_index(*, &y)
         # insert the cassette recording everything what happens outside the tests cases
         VCR.insert_cassette("#{self.class.metadata[:description]}/#{self.class.metadata[:vcr_all_cassette] || 'all'}")
 
-        # avoid polling idle time by overriding sleep
-        if skip_sleep
+        unless VCR.current_cassette.recording?
+          # avoid polling idle time by overriding sleep
           module Kernel
             alias :old_sleep :sleep
             def sleep(n)
               n
             end
           end
+
+          # make vcr tests work without password
+          module GoodData::Helpers
```

Comment:
warning: Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style. (rubocop)